.ci/dockerfiles/Dockerfile.base

```
@@ -36,6 +36,16 @@ LABEL maintainer="NVIDIA NIXL Team"
 LABEL description="Test environment for NIXL development"
 LABEL version="1.0"
 
+# Use the internal Ubuntu mirror (public archive/ports.ubuntu.com is unreliable).
+RUN . /etc/os-release && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)